README.md

```diff
@@ -33,6 +33,24 @@
 
 </div>
 
+# Table of Contents  
+
+- [📮 About](#-about)
+- [🕹 Project Motivation](#-project-motivation)
+- [✂ Current Features](#-current-features)
+- [:mountain: GUI clients](#mountain-gui-clients)
+- [⌚ Current Plugins](#-current-plugins)
+- [⚡ Quickstart](#-quickstart)
+- [💻 Seeing The Bot In Action](#-seeing-the-bot-in-action)
+- [Docker](#docker)
```

Comment:
Shall we use :whale: `:whale:` or :whale2: `:whale2:`?

Reply:
I think :whale: is nice because of the water.